docs/uma-tokenholders/ulabs-liquiditymining.md

```diff
@@ -0,0 +1,23 @@
+---
+title: uLABs Liquidity Mining Program
+sidebar_label: Liquidity Mining
+---
+
+The Liquidity Mining program was [moved under](https://twitter.com/UMAprotocol/status/1324199135080148993) the umbrella of the developer mining, with uLABs as the developer responsible for distributing rewards accrued by 
+yield dollar tokens(YD-ETH-MAR21 & YD-BTC-MAR21). 
+
+## YD-ETH-MAR21 
```

Comment:
I don't think it's a bad idea to have something similar to this somewhere as a stop-gap before we have an application for it.

@Tommy1231232 I know we try to keep our website non-LM focused. Think a slightly edited version of this could work until then?